shared/js/background/classes/tracker-stats.js

```diff
@@ -0,0 +1,192 @@
+import z from 'zod'
+
+/**
+ * A single place to collect information about the number of requests
+ * we have blocked, grouped by the owner's display name.
+ */
+export class TrackerStats {
+    maxAgeMs = 1000 * 60 * 60 // 1hr
```

Comment:
Do we only keep tracker counts for 1 hour? Won't this then be mostly empty most of the time for light users, and at the beginning of a new session?

Reply:
Yep - that's the intention though. :)